reporters_db/data/reporters.json

```diff
@@ -22505,9 +22593,17 @@
             "editions": {
                 "Wall.": {
                     "end": "1874-12-31T00:00:00",
+                    "regexes": [
+                        "$full_cite",
+                        "$reporter,? (p.)? $page"
```

Comment:
do we have an example without the page number?

Reply:
I couldn't find one, should I leave the "p." fixed instead of optional?

Reply:
Drop the optional grouping.  Is this a nominative volume reporter?